### PR TITLE
Change the statistics type from float64 to uint64

### PR DESCRIPTION
**Pull request by aswinsuryan**
The code generator does not support float64 and hence updating
the API to use uint64

Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

**Comment by tpantelis** [comment]
Can you provide more detail? What code generator and how did it fail?

**Comment by aswinsuryan** [comment]
> Can you provide more detail? What code generator and how did it fail?

cd vendor/github.com/submariner-io/submariner && controller-gen "crd:crdVersions=v1,trivialVersions=false" paths="./..." output:crd:artifacts:config=../../../../deploy/submariner/crds
/go/src/github.com/submariner-io/submariner-operator/vendor/github.com/submariner-io/submariner/pkg/apis/submariner.io/v1/types.go:89:13: unsupported type "float64"

This is the error it shows.

**Review by SteveMattar — APPROVED** [review]

**Comment by tpantelis** [comment]
Discussed  on chat - we should just change them to uint64.

**Comment by aswinsuryan** [comment]
> Discussed on chat - we should just change them to uint64.
done.

**Review by SteveMattar — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

File: pkg/apis/submariner.io/v1/types.go [hunk 1]
```diff
@@ -86,11 +86,11 @@ type GatewayStatus struct {
 // LatencySpec describes the round trip time information in miliseconds for a packet
```
Comment (skitt) [inline pkg/apis/submariner.io/v1/types.go:86]: To preserve the granularity, shouldn’t the values use a finer unit, say microseconds or nanoseconds?

**Review by skitt — CHANGES_REQUESTED** [review]

File: pkg/apis/submariner.io/v1/types.go (hunk 1 above)
Comment (aswinsuryan) [reply pkg/apis/submariner.io/v1/types.go:86]: Ok the time period gives it in nano seconds, I will change it to nano seconds.

**Review by skitt — APPROVED** [review]

**Pull request merged**